src/services/api/serverless.yml

```diff
@@ -107,7 +107,7 @@ functions:
       osDomain: ${param:osDomain}
     events:
       - http:
-          path: /search
+          path: /search/{index}
```

Comment:
🔥 Love that we're keeping the endpoint dynamic params pattern from Actions. Keeps endpoints uniform!